src/app-layer-dns-udp.c

```diff
@@ -316,6 +316,7 @@ static int DNSUDPResponseParse(Flow *f, void *dstate,
             tx->recursion_desired = 1;
         }
 
+        tx->flags = ntohs(dns_header->flags);
```

Comment:
should this also be set for DNS/TCP?

cc: @jasonish

Reply:
Right, I believe so. I am doing the update within app-layer-dns-tcp.c file (line 513, adding this code tx->flags = ntohs(dns_header->flags); ). 
So far I do not have any dns over tcp pcap sample file. I will try to get some in my lab to verify this.

Reply:
I've emailed you a pcap.

Reply:
ok received and used to check the update for TCP. It works fine.